### PR TITLE
DELLEMC Networker B&R Plugin Documentation

### PR DESCRIPTION
**Pull request by fermosan**

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Comment by fermosan** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@fermosan why do we need the `.idea` and `.tx` directories? can these be removed?

**Comment by fermosan** [comment]
@DaanHoogland We should be ok now. Let me know if you want any changed in the documentation.

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ build/
 source/_build/
 /.idea/
 /.tx/
+/.idea
+/.tx
```
Comment (DaanHoogland) [inline .gitignore:7]: are these all needed?
I think 
```suggestion
.idea
.tx
```
should do

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Comment by fermosan** [comment]
@blueorangutan docbuild

File: source/adminguide/networker_plugin.rst [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+.. |BnR-Networker-Cluster-Clients-overview.jpg| image:: /_static/images/BnR-Networker-Cluster-Clients-overview.jpg
+   :alt: Cluster Clients Overview.
+   :width: 300 px
+
+
+
+
+
+
+
```
Comment (DaanHoogland) [inline source/adminguide/networker_plugin.rst:221]: extra lines? are these needed?

Comment (DaanHoogland) [reply source/adminguide/networker_plugin.rst:221]:
```suggestion
```

**Pull request merged**